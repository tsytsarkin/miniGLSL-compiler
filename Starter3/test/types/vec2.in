{
    vec2 i;
    vec2 j = vec2(1.23, 2.45);
    j[0] = 3.456;
}
