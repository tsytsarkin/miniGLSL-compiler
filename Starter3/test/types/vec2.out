  1: {
  2:     float i;
  3:     vec2 j = vec2(1.23, 2.45);
  4:     j[0] = 3.456;
  5:     i = j[1];
  6: }
 (SCOPE (DECLARATIONS (DECLARATION i float) (DECLARATION j vec2 (CALL vec2 1.230000 2.450000))) (STATEMENTS (ASSIGN float (INDEX float j 0) 3.456000) (ASSIGN float i (INDEX float j 1))))
